src/utilities/omit.js

```diff
@@ -0,0 +1,5 @@
+export default function omit(object = {}, ...keys) {
+  const copy = Object.assign({}, object);
+  keys.forEach(key => delete copy[key]);
+  return copy;
+}
```

Comment:
as far as you do copy on your own. its better to create fresh copy and then add every non-omitted key-val into copy from original object